docs/playbooks/lite-assist-quickburn.md

```diff
@@ -0,0 +1,125 @@
+# MAS Assist Service on DevIT Quickburn
+
+This master playbook will drive the following playbooks in sequence:
+
+- [Provision & setup Quickburn](ocp.md#quickburn) (25 minutes)
+- [Prepare the OCS cluster ](../roles/ocp_setup_ocs.md) (10 minutes)
+- Install MAS dependencies:
+    - [Install MongoDb](dependencies.md#install-mongodb-ce) (10 minutes)
+    - [Install SLS](dependencies.md#install-sls) (10 minutes)
+- Install & configure MAS:
+    - [Configure Cloud Internet Services integration](mas.md#cloud-internet-services-integration) (Optional, 1 minute)
+    - Generate MAS Workspace Configuration (1 minute)
+    - [Install & configure MAS](mas.md#install-mas) (15 minutes)
+- [Configure and prepare object storage instance](../roles/cos_setup.md)
+- Install and prepare discovery instance
+    - [Install discovery Service](cp4d.md#install-services-discovery)
+    - [Create Disocvery instance](cp4d.md#create-discovery-instance)
+- Install Assist and Configure Assist Workspace
+    - [Refer to Suite App Assist](../roles/suite_app_install.md) (20 Minutes)
+    - [Refer to Suite App Configure](../roles/suite_app_configure.md)  (10 Minutes)
+  
+
+All timings are estimates, see the individual pages for each of these playbooks for more information.  Due to the size limtations of QuickBurn clusters a full MAS stack is not possible.
+
+## Required environment variables
+- `FYRE_USERNAME`
+- `FYRE_APIKEY`
+- `FYRE_PRODUCT_ID`
+- `CLUSTER_NAME` The name to assign to the new Quickburn cluster
+- `CLUSTER_TYPE` The cluster type. Should be set to `quickburn`
+- `MAS_INSTANCE_ID` Declare the instance ID for the MAS install
+- `MAS_ENTITLEMENT_KEY` Lookup your entitlement key from the [IBM Container Library](https://myibm.ibm.com/products-services/containerlibrary)
+- `MAS_CONFIG_DIR` Directory where generated config files will be saved (you may also provide pre-generated config files here)
+- `SLS_LICENSE_ID` The license ID must match the license file available in `$MAS_CONFIG_DIR/entitlement.lic`
```

Comment:
Same issue as the other md file and the need for SLS_LICENSE_FILE

Reply:
updated